app/views/shared/_webchat.html.erb

```diff
@@ -1,6 +1,11 @@
 <span class="js-webchat"
   data-availability-url="<%= @content_item.webchat_availability_url %>"
   data-open-url="<%= @content_item.webchat_open_url %>">
+  <% if @content_item.base_path == '/government/organisations/hm-revenue-customs/contact/tax-credits-enquiries' %>
+    Advisers can only talk to you about Child Tax Credit and Working Tax Credit.
+  <% end %>
+  <br/>
```

Comment:
Should the line breaks be in the `if` statement?

Reply:
Fixed. 👍